### PR TITLE
Tech and title cards

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary
-Added tech component.
-Added title component.
-Flexing for tech components.

## Screenshot
screenshot from 2018-08-10 11-18-42

## Trello Card
-Tecnología
-Page Titles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+    <div class="tech-set">
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/react_icon.svg" alt="React logo"></img>
+        <h1 class="tech-title">React JS</h1>
+        <strong class="tech-text">Release: 2013</strong>
+        <p class="tech-text">React is a declarative, component-based
+          JavaScript library for building user interfaces.</p>
+      </div>
+    </div>
+
```
Comment (FrankIglesias) [inline index.html]: delete this two lines

File: scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import 'equalizer';
+@import 'styles';
+@import 'titles';
+@import 'tech';
+
+.center-content {
```
Comment (FrankIglesias) [inline scss/application.scss]: move this to a layout scss

File: scss/styles.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
-.title {
-  color: red;
-}
+$blue: #00ADEE;
```
Comment (FrankIglesias) [inline scss/styles.scss]: move this to colors.scss file

File: scss/styles.scss [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$text-big: 24px;
```
Comment (FrankIglesias) [inline scss/styles.scss]: move this to a text file

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this line

File: scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 280px;
+  margin: 20px;
+  max-height: 100%;
+  left: 8px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this

File: scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  top: 8px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: and this

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  width: 190px;
+  max-width: 100%;
+  padding: 5px 30px 5px 25px;
+  position:relative;
```
Comment (FrankIglesias) [inline scss/tech.scss]: add space before 'relative'

File: scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  text-align: center;
+}
+
+.tech:before {
```
Comment (FrankIglesias) [inline scss/tech.scss]: nest this inside tech

File: scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+  content: '';
+  border: 2px solid $green;
+  border-radius: inherit;
```
Comment (FrankIglesias) [inline scss/tech.scss]: why this?

File: scss/tech.scss [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+  height: inherit;
+  position: absolute;
+  left: -10px;
+  top: -12px;
+  width: inherit;
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this line

File: scss/tech.scss [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+}
```
Comment (FrankIglesias) [inline scss/tech.scss]: add a z-index prop

File: scss/tech.scss [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+.tech-logo{
+  display: block;
+  margin: 5px auto 5px auto;
+  height: 100px;
+  max-height: 100%;
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this

File: scss/tech.scss [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+  width: 100px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: only fix one axis... i'd recommend only width size

File: scss/tech.scss [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+  max-width: 100%;
+}
+
+.tech-title {
+ font-family: 'Roboto';
```
Comment (FrankIglesias) [inline scss/tech.scss]: define this font-family in a more generic element, like body

File: scss/tech.scss [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+  margin-top: 10px;
+}
+
+.tech-text {
+  margin-top: 15px;
+  font-family: 'Roboto';
```
Comment (FrankIglesias) [inline scss/tech.scss]: this is unnesesary

File: scss/tech.scss [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+.tech-set {
+  align-content: flex-start;
+  display:flex;
+  width: 100%;
+  height: 100%;
+  margin: auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this line

File: scss/tech.scss [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+.tech-set {
+  align-content: flex-start;
+  display:flex;
+  width: 100%;
+  height: 100%;
+  margin: auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
```
Comment (FrankIglesias) [inline scss/tech.scss]: this is the default prop

File: scss/tech.scss [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+  font-size: $text-small;
+}
+
+.tech-set {
+  align-content: flex-start;
+  display:flex;
+  width: 100%;
+  height: 100%;
+  margin: auto;
+  flex-direction: row;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.tech-text {
+  margin-top: 15px;
+  font-family: 'Roboto';
+  font-size: $text-small;
+}
+
+.tech-set {
+  align-content: flex-start;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/titles.scss [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+$title-decoration-width: 70px;
+
+.title {
+  color:black;
+  font-size: $text-big;
+  margin: 10px;
+  padding-bottom: 35px;
+  padding-top: 35px;
+  padding-left: 45px;
+
```
Comment (FrankIglesias) [inline scss/titles.scss]: delete line

File: scss/titles.scss [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.title:before {
+  content:'';
+  background: url(assets/images/triangle.svg);
```
Comment (FrankIglesias) [inline scss/titles.scss]: i think you missed the comma

File: scss/titles.scss [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+  color:black;
+  font-size: $text-big;
+  margin: 10px;
+  padding-bottom: 35px;
+  padding-top: 35px;
+  padding-left: 45px;
+
+}
+
+.title:before {
```
Comment (FrankIglesias) [inline scss/titles.scss]: nest this class into title

File: scss/titles.scss [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+  content:'';
+  background: url(assets/images/triangle.svg);
+  background-repeat: no-repeat;
+  display: block;
+  height: $tile-decoration-height;
+  margin-left: 15px;
+  margin-top:10px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: add a space before 10px

File: scss/titles.scss [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+}
+
+.title:before {
+  content:'';
+  background: url(assets/images/triangle.svg);
+  background-repeat: no-repeat;
+  display: block;
+  height: $tile-decoration-height;
+  margin-left: 15px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: use a shorthand for margins

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by mikolaslorant** [comment]
Already corrected changes

File: scss/tech.scss [hunk 26]
```diff
@@ -0,0 +1,50 @@
...
+  margin-top: 10px;
+}
+
+.tech-text {
+  margin-top: 15px;
+  font-size: $text-small;
+}
+
+.tech-set {
+  display:flex;
```
Comment (FrankIglesias) [inline scss/tech.scss]: add space after comma

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/titles.scss [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+.title {
+  color:black;
+  font-size: $text-big;
+  margin: 10px;
+  padding-bottom: 35px;
+  padding-top: 35px;
+  padding-left: 45px;
+
+  &::before {
+    content:'';
```
Comment (FrankIglesias) [inline scss/titles.scss]: same

**Comment by mikolaslorant** [comment]
Corrected new requested changes

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**